packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/customerForm.js

```diff
@@ -84,12 +77,8 @@ const CustomerForm = props => {
         : 'Add';
 
     const submitButtonProps = {
-        classes: {
-            root_normalPriority: classes.submit,
-            root_highPriority: classes.submit_update
-        },
         disabled: isSaving,
-        priority: isUpdate ? 'high' : 'normal',
```

Comment:
I doubt this was an accident originally. Does the style guide call for all proceed/submit buttons to be high priority?

Reply:
Both 'Update' and 'Add' are considered primary actions in the address modal, I modified this logic to the `Save and Continue` button is the correct `normal` priority if the authenticated user doesn't have any saved addresses :)